projects/angular/src/data/datagrid/datagrid.html

```diff
@@ -22,7 +22,7 @@
                     *ngIf="selection.selectionType === SELECTION_TYPE.Multi"
                     (keydown.space)="allSelected = !allSelected; $event.preventDefault()"
                   >
-                    <div class="clr-checkbox-wrapper">
+                    <div *ngIf="!hasVirtualScroller" class="clr-checkbox-wrapper">
```

Comment:
Clicking the header throws error from 'datagrid-selection-cell.directive.ts' because we are still tracking click events by the .div abbove

Reply:
Corrected.